### DISCUSSION TITLE
Unable to upload Excel file " to table  in database "examples". Error message: Excel file format cannot be determined, you must specify an engine manually.

### DISCUSSION THREAD

**Discussion by Raj2598** [apache/superset#22556] [Q&A / Help]
Hello, I am new to superset and I am trying to make charts/KPIs in Superset by uploading Excel files, but I face this error " Unable to upload Excel file "filename.xls" to table "tableName" in database "examples". ****Error message: Excel file format cannot be determined, you must specify an engine manually."**** Anyone has ever faced similar issue can help me regarding this error? Thanks.

**Comment by duniang818** [comment]
do you have resolved ? 
how and can you share the solution?
